Origin: DHI-internal (package build compatibility)
Description: Drop janusgraph-dist javadoc classifier dependency and unpack assembly;
 DHI packages strip javadocs from the staged tree and builds with -Dmaven.javadoc.skip=true.
Upstream: https://github.com/JanusGraph/janusgraph/blob/v1.2.0/janusgraph-dist/pom.xml
Forwarded: no

diff --git a/janusgraph-dist/pom.xml b/janusgraph-dist/pom.xml
--- a/janusgraph-dist/pom.xml
+++ b/janusgraph-dist/pom.xml
@@ -370,12 +370,6 @@
         <profile>
             <id>janusgraph-release</id>
             <dependencies>
-                <dependency>
-                    <groupId>${project.groupId}</groupId>
-                    <artifactId>janusgraph-core</artifactId>
-                    <version>${project.version}</version>
-                    <classifier>javadoc</classifier>
-                </dependency>
             </dependencies>
             <build>
                 <plugins>
@@ -610,21 +604,6 @@
                                     </formats>
                                 </configuration>
                             </execution>
-                            <execution>
-                                <id>unpack-janusgraph-core-javadocs</id>
-                                <phase>package</phase>
-                                <goals>
-                                    <goal>single</goal>
-                                </goals>
-                                <configuration>
-                                    <attach>false</attach>
-                                    <finalName>javadocs</finalName>
-                                    <appendAssemblyId>false</appendAssemblyId>
-                                    <descriptors>
-                                        <descriptor>${assembly.descriptor.dir}/javadocs.xml</descriptor>
-                                    </descriptors>
-                                </configuration>
-                            </execution>
                         </executions>
                     </plugin>
 
